pkg/connector/handletwit.go

```diff
@@ -247,3 +248,18 @@ func (tc *TwitterClient) updateTwitterReadReceipt(inbox *response.TwitterInboxDa
 		tc.participantCache[conversationID] = conversation.Participants
 	}
 }
+
+func (tc *TwitterClient) updateTwitterUserInfo(inbox *response.TwitterInboxData) {
+	ctx := context.Background()
```

Comment:
New contexts should always have a logger attached

```suggestion
	ctx := tc.userLogin.Log.With().Str("action", "update user info").Logger().WithContext(context.Background())
```